agent/installer/installer.go

```diff
@@ -24,6 +25,8 @@ const (
 	ErrBundleUninstall   = Error("Error uninstalling bundle")
 )
 
+var preRequisitePackages = []string{"socat", "ebtables", "ethtool", "conntrack"}
```

Comment:
futurescope: @anusha94 / @raunaksingwi if we plug this list out i feel there is potential of a generic lib that can be used across projects 😄

Reply:
@pshail Can you say more? Plug out where? Coming from a config file of sorts?

Reply:
Yup - basically plug-out the requisite packages